images/dcos/Dockerfile

```diff
@@ -1,25 +1,32 @@
-FROM busybox:ubuntu-14.04
+FROM alpine:3.2
```

Comment:
An alternative would be busybox with the nginx binary from the Ubuntu packages. No idea how man zillions of dependencies this means.